server/plan.go

```diff
@@ -40,5 +40,10 @@ func (s *Server) registerPlanRoutes(g *echo.Group) {
 }
 
 func (s *Server) feature(feature api.FeatureType) bool {
-	return api.FeatureMatrix[feature][s.plan]
+	plan := api.TEAM
```

Comment:
The default plan type is `TEAM`, to align with our current plan feature. Will change to `FREE` later